roadmap/implementers-guide/src/node/disputes/dispute-coordinator.md

```diff
@@ -4,10 +4,324 @@ The coordinator is the central subsystem of the node-side components which parti
 wraps a database, which used to track all statements observed by _all_ validators over some window
 of sessions. Votes older than this session window are pruned.
 
-This subsystem will be the point which produce dispute votes, either positive or negative, based on
-locally-observed validation results as well as a sink for votes received by other subsystems. When
-a statement import makes it clear that a dispute has been raised (there are opposing votes for a
-candidate), the dispute coordinator will make sure the local node will participate in the dispute.
+In particular the dispute-coordinator is responsible for:
+
+- Ensuring that the node is able to raise a dispute in case an invalid candidate
+  is found during approval checking.
+- Ensuring malicious approval votes will be recorded, so nodes can get slashed
+  properly.
+- Coordinating actual participation in a dispute, ensuring that the node
+  participates in any justified dispute in a way that ensures resolution of
+  disputes on the network even in the case of many disputes raised (flood/DoS
+  scenario).
+- Provide an API for chain selection, so we can prevent finalization of any
+  chain which has included candidates for which a dispute is either ongoing or
+  concluded invalid and avoid building on chains with an included invalid
+  candidate.
+- Provide an API for retrieving (resolved) disputes, including all votes, both
+  implicit (approval, backing) and explicit dispute votes. So validators can get
+  rewarded/slashed accordingly.
+
+## Ensuring That Disputes Can Be Raised
+
+If a candidate turns out invalid in approval checking, the `approval-voting`
+subsystem will try to issue a dispute. For this, it will send a message
+`DisputeCoordinatorMessage::IssueLocalStatement` to the dispute coordinator,
+indicating to cast an explicit invalid vote. It is the responsibility of the
+dispute coordinator on reception of such a message to create and sign that
+explicit invalid vote and  trigger a dispute if none is already
```

Comment:
Nit: No dispute is ongoing for *the same* dispute ?

Reply:
Yep, thanks Tara! will clarify!

Reply:
dispute for the same candidate to be precise. Clarified.